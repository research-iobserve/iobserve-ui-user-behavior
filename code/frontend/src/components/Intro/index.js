// @flow

import React from "react";
import { Jumbotron, Container } from "reactstrap";
import styled from "styled-components";

import H1 from "../../components/H1";

const StyledJumbotron = styled(Jumbotron)`
  background-color: #fff;
  margin-bottom: 0;
  padding-bottom: 6rem;
  padding-top: 6rem;
`;

function Intro() {
  return (
    <StyledJumbotron>
      <Container>
        <H1>User Model Behavior Visualization</H1>
        <p className="lead">
          Below is a list of currently available user behavior graphs. Please select one to inspect it.
        </p>
      </Container>
    </StyledJumbotron>
  );
}

export default Intro;
