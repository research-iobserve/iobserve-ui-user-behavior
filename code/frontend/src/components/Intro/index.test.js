// @flow

import React from "react";
import renderer from "react-test-renderer";

import Intro from "./index";

describe("<Intro />", () => {
  it("should render", () => {
    const tree = renderer.create(<Intro />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
