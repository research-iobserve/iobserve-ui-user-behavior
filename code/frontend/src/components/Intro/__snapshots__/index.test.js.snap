exports[`<Intro /> should render 1`] = `
<div
  className="gFpPfz jumbotron">
  <div
    className="container">
    <h1
      className="fdavpS">
      User Model Behavior Visualization
    </h1>
    <p
      className="lead">
      Below is a list of currently available user behavior graphs. Please select one to inspect it.
    </p>
  </div>
</div>
`;
